============================= test session starts ==============================
tests/test_crypto.py ...........EE.....EE.................................................................s.s.......................................................................................................................FFs..........................F......................
tests/test_debug.py .
tests/test_rand.py ....
tests/test_ssl.py .............s.........s.......s.....s..s..s..s.............s.ss...........s...............s..FF..................................s...FF.......s....................s.....sF.............s...............................................
tests/test_tsafe.py .
tests/test_util.py .
==================================== ERRORS ====================================
=================================== FAILURES ===================================
tests/test_crypto.py:3196: AttributeError
tests/test_crypto.py:3219: AttributeError
tests/test_crypto.py:3623: Failed
tests/test_ssl.py:1616: 
tests/test_ssl.py:1625: 
tests/test_ssl.py:2433: 
tests/test_ssl.py:2445: 
tests/test_ssl.py:2959: AttributeError
========== 8 failed, 467 passed, 20 skipped, 4 error in x seconds ==========
